addons/interaction/functions/fnc_canPullOutBody.sqf

```diff
@@ -18,6 +18,9 @@
 
 params ["_body", "_unit"];
 
+// Defer to ACE Medical's unload patient if present
```

Comment:
I think we still need this for pulling out dead bodies

Reply:
UnloadPatient handles it now.

Reply:
Should have been handled by #9281